tests/ConnectionTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Db\Mysql\Tests;
+
+use Yiisoft\Db\Drivers\Connection;
+use Yiisoft\Db\Helper\Dsn;
+use Yiisoft\Db\Tests\ConnectionTest as AbstractConnectionTest;
+
+class ConnectionTest extends AbstractConnectionTest
+{
+    protected ?string $driverName = 'mysql';
+
+    public function testDsnHelper(): void
```

Comment:
Can we transfer this testcase to parent class and fill in subclass only with different paramaters?

Reply:
I tell you, there are currently many tests that are marked as markTestSkipped, precisely because all implementations do not support it, the idea is that each implementation has its test without skipping any.